src/controllers/movies.js

```diff
@@ -0,0 +1,86 @@
+const {
+  getAllMoviesDb,
+  createMovieDb,
+  getMovieByIdDb,
+  updateMovieByIdDb,
+} = require("../domains/movies.js");
+
+const getAllMovies = async (req, res) => {
+  try {
+    const allMovies = await getAllMoviesDb(req.query);
+
+    return res.status(200).send({ movies: allMovies });
+  } catch (e) {
+    console.error(e.message);
+    return res.status(500).send({ error: e.message });
+  }
+};
+
+const getMovieById = async (req, res) => {
+  const id = Number(req.params.id);
+
+  const foundMovie = await getMovieByIdDb(id);
+
+  if (foundMovie) {
+    return res.status(200).send({ movie: foundMovie });
+  }
+
+  return res.status(404).send({ error: "No movie found with provided ID" });
+};
+
+const createMovie = async (req, res) => {
+  try {
+    const { title, runtimeMins } = req.body;
+
+    if (!title && !runtimeMins) {
+      return res.status(400).send({ error: "Missing fields in request body" });
```

Comment:
I'd move your error strings to a central location so you don't have the same string in multiple places. Easier to maintain that way

Reply:
Thanks for the review, good suggestion!